components/swatch/stories/swatch.stories.js

```diff
@@ -49,23 +57,193 @@ export default {
+ */
 export const Default = SwatchGroup.bind({});
 Default.args = {};
 
+// ********* DOCS ONLY ********* //
+/**
+ * Use the other sizes sparingly; they should be used to create a hierarchy of importance within the page.
+ */
+export const Sizing = (args, context) => Sizes({
+	Template,
+	withHeading: false,
+	...args,
+}, context);
+Sizing.tags = ["!dev"];
+Sizing.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * A swatch in a disabled state shows that the swatch exists, but is not available in that circumstance. Even though swatches can have a disabled state, hide unavailable swatches when possible to reduce visual clutter and ease cognitive load. Only show disabled swatches if hiding them would cause confusion to your users.
+ */
+export const Disabled = (args, context) => Sizes({
+	Template: DisabledGroup,
+	withHeading: false,
+	...args,
+}, context);
+Disabled.args = {
+	isDisabled: true,
+};
+Disabled.tags = ["!dev"];
+Disabled.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * Default rounding and full rounding are usually used when a swatch is presented by itself near other components. A rounding of “none” is used in a swatch group to help minimize the Hermann grid illusion that happens at the intersections of white space in the group.
+ */
+export const Rounding = RoundingGroup.bind({});
+Rounding.tags = ["!dev"];
+Rounding.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * A swatch can have a selected state to allow for selection. This is often used in a [swatch group](?path=/docs/components-swatch-group--docs).
+ */
+export const Selected = Template.bind({});
+Selected.args = {
+	isSelected: true,
+};
+Selected.tags = ["!dev"];
+Selected.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * By default, swatches have a border. However, when swatches are used within a swatch group, there are additional border considerations. 
+ * - When color swatches are used in a [swatch group](?path=/docs/components-swatch-group--docs), they typically have the `.spectrum-Swatch--noBorder` class.
+ * - When and only when color swatches used in a swatch group have low contrast (below 3:1 contrast with the background), those swatches will have a less prominent border compared to the swatch component when used by itself. They individually use the `.spectrum-Swatch--lightBorder` class.
+ */
+export const Border = BorderGroup.bind({});
+Border.tags = ["!dev"];
+Border.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/** 
+ * Swatches can also have a rectangle shape with an aspect ratio of 2:1. The square shape is the default and is used in swatch groups (e.g., a palette of colors).
+ */
+export const Rectangle = Template.bind({});
+Rectangle.args = {
+	isRectangle: true,
+};
+Rectangle.tags = ["!dev"];
+Rectangle.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+export const Nothing = (args, context) => Sizes({
```

Comment:
Hmmm, I could go either way. If you change the name, maybe mention a reference to "Nothing" in the description somewhere so that it's a little clearer they're meant to be the same if someone encounters "Nothing" somewhere else.

Reply:
I did end up changing it to Empty. But great point about calling out "nothing" still- the class name is `is-nothing` so that's added now as well: https://github.com/adobe/spectrum-css/pull/2925/commits/c89f710f201aaac8080cf0e2db126126a9cb03ef